### ISSUE TITLE
Azure provider fails installing the Azure agent for Linux

### ISSUE THREAD

**Issue by tomconte** [andsens/bootstrap-vz#231]
The process fails with the error below (incorrectly looking in 'system' section instead of 'provider'). Working on a fix.

```
'waagent'
Traceback (most recent call last):
  File "/home/azureuser/bootstrap-vz/bootstrapvz/base/main.py", line 109, in run
    tasklist.run(info=bootstrap_info, dry_run=dry_run)
  File "/home/azureuser/bootstrap-vz/bootstrapvz/base/tasklist.py", line 43, in run
    task.run(info)
  File "/home/azureuser/bootstrap-vz/bootstrapvz/providers/azure/tasks/packages.py", line 46, in run
    if info.manifest.system['waagent'].get('conf', False):
KeyError: 'waagent'
```

**Comment by andsens** [comment]
Woops. That one slipped through the cracks when I refactored.
> Working on a fix.

Great! I'll hold off on fixing it myself then :-)

**Comment by tomconte** [comment]
Fixed in b7243449f5264cf0da752e626a5a5392a7f39dc0
